rfcs/050-concepts-api/README.md

```diff
@@ -0,0 +1,200 @@
+# RFC 050: Design considerations for the concepts API
+
+This RFC collects some initial thinking on how we might represent concepts in the catalogue API.
+It's a starting point for discussions; not a final design.
+
+
+
+## Goals
+
+*   We want to be able to create per-person and per-subject pages on the website.
+    The data on these pages should come from the new concepts API.
+
+*   These pages should contain enough context to distinguish two people/subjects (e.g. whether "Charles Darwin" is the famous naturalist, his uncle, his son, or his grandson).
+
+*   We're not trying to replicate Wikidata -- the APIs shouldn't return large amounts of information we aren't going to use on wellcomecollection.org.
+
+*   Any new APIs should be similar to the existing works and images APIs.
+    We'll use the same conventions for filters, aggregations, including optional fields, and so on.
+
+
+
+## Endpoint #1: retrieve a single concept
+
+This is the single concept endpoint, which is used to render a page for an individual subject or person:
+
+```
+GET /concepts/azxzhnuh
+{
+  "id": "azxzhnuh",
+  "identifiers": [
+    {
+      "identifierType": "lc-names",
+      "value": "n12345678",
+      "type": "Identifier"
+    },
+    ...
+  ],
+
+  "label": "Florence Nightingale",
+  "alternativeLabels": [
+    "The Lady with the Lamp",
+  ]
+
+  "type": "Person|Subject|Organisation|Place"
+
+  // Everything above here is stuff I'm pretty sure we'll need;
+  // everything below it is more nebulous and more likely to change
+  // in the final API.
+
+  "description": "Florence Nightingale was an English social reformer, statistician and the founder of modern nursing. Nightingale came to prominence while serving as a manager and trainer of nurses during the Crimean War, in which she organised care for wounded soldiers at Constantinople.",
+
+  // not locations
+  "urls": [
+    "https://en.wikipedia.org/wiki/Florence_Nightingale"
+  ],
+
+  // cf productionEvent?
+  "dates": [ { date, meaning } ]
+  "places": [ { place, meaning } ]
+  "birthDate/place"
+  "deathDate/place"
+
+  "thumbnail": { … },
+
+  "connectedConcepts": [
```

Comment:
I can see us breaking this out a bit into multiple kinds of relationship. However, I don't see concept relationships happening for any early version so I wouldn't worry too much for now.